bin/export.py

```diff
@@ -122,9 +127,6 @@ def build_problem_zip(problem: Problem, output: Path):
     if problem.custom_output:
         files.append(("output_validator/**/*", True))
 
-    if config.args.kattis:
-        files.append(("input_validators/**/*", True))
-
     message("preparing zip file content", "Zip", problem.path, color_type=MessageType.LOG)
 
     # prepare files inside dir
```

Comment:
(comment should actually be at line 136, but that's outside the diff)

The comment says:

> `# For Kattis, prepend the problem shortname to all files.`

The spec says at https://icpc.io/problem-package-format/spec/2023-07-draft.html#general-requirements:

> The package must consist of a single directory containing files as described below. [...] the package can be a ZIP-compressed archive of such a directory with identical base name and extension [...] `.zip`.

So I guess we should _always_ prepend the problem short name this now (except for legacy, simply because DOMjudge doesn't support the new format yet)?